backend/cmd/server/servicemanager.go

```diff
@@ -43,10 +44,18 @@ import (
 	"github.com/asgardeo/thunder/internal/userschema"
 )
 
+var ObservabilitySvc observability.ObservabilityServiceInterface
```

Comment:
We're not assigning any value/ service to this variable right?
If this is not needed, can remove code in main.go as well